### PR TITLE
Jakarta EL 5.0 plan review

### PR DESCRIPTION
**Pull request by markt-asf**
## Plan Review PR template
When creating a specification project plan review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version.
- [x] An index page wombat/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
- [x] Title should be on the form "Jakarta Wombat X.Y (under development)"
- [x] Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [x] Includes a detailed description of any backward incompatibility (Mark with **N/A** and check if none)
- [x] Declares optional features if there are any (Mark with **N/A** and check if none)
- [x] Includes minimum Java SE version
- [x] Describe any updates or required changes including splitting out the TCK (Mark with **N/A** and check if not planned)
- [x] Link to updated release record

### CODE REVIEW

File: expression-language/5.0/_index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+* [Jakarta Expression Language 5.0 Release Record](https://projects.eclipse.org/projects/ee4j.el/releases/5.0.0)
+    * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
+* [Jakarta Expression Language 5.0 Specification Document](./jakarta-expression-language-spec-4.0.pdf) (PDF)
+* [Jakarta Expression Language 5.0 Specification Document](./jakarta-expression-language-spec-4.0.html) (HTML)
+* [Jakarta Expression Language 5.0 Javadoc](./apidocs)
+* [Jakarta Expression Language 5.0 TCK](https://download.eclipse.org/jakartaee/expression-language/5.0/jakarta-expression-language-tck-5.0.0.zip)([sig](https://download.eclipse.org/jakartaee/expression-language/5.0/jakarta-expression-language-tck-5.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/expression-language/5.0/jakarta-expression-language-tck-5.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* Maven coordinates
+     * [jakarta.el:jakarta.el-api:jar:5.0.0](https://repo1.maven.org/maven2/jakarta/el/jakarta.el-api/5.0.0/)
```
Comment (Pandrex247) [inline expression-language/5.0/_index.md:15]: I'm not sure you need these bits yet - it's just a plan review at this stage.
(You're also pointing at the 4.0 spec!)

Can you remove them please?

File: expression-language/5.0/_index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+* New tests for any new features
+* Changes to support the removal of the java.desktop dependency (if implemented)
+
+# Release Record
+[Release Record](https://projects.eclipse.org/projects/ee4j.el/releases/5.0.0)
+
+# Compatible Implementations
+
+* [Glassfish](https://repo1.maven.org/maven2/org/glassfish/jakarta.el/5.0.0/)
+* [Apache Tomcat 10.1.0](https://tomcat.apache.org/download-101.cgi)
```
Comment (Pandrex247) [inline expression-language/5.0/_index.md:48]: Same as above - don't think you need this bit for a plan review.

**Review by Pandrex247 — COMMENTED** [review]
Just a couple tweaks and it should be good to go.

File: expression-language/5.0/_index.md (hunk 1 above)
Comment (markt-asf) [reply expression-language/5.0/_index.md]: Done.

**Review by Pandrex247 — APPROVED** [review]
LGTM, will begin the ballot

File: expression-language/5.0/_index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+|Fujitsu                |         |         |          |
+|IBM                    |         |         |          |
+|Oracle                 |         |         |          |
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
```
Comment (kwsutter) [inline expression-language/5.0/_index.md:60]: You are missing the reference for the Spec mailing list ballot thread:  `This ballot was conducted on the public e-mail list [jakarta.ee-spec@eclipse.org](). This ballot thread begins [here]().`

File: expression-language/5.0/_index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+|-----------------------|---------|-----|----------|
+|Fujitsu                |         |     |          |
+|IBM                    |         |     |          |
+|Oracle                 |         |     |          |
+|Payara                 |         |     |          |
+|Red Hat                |         |     |          |
+|Tomitribe              |         |     |          |
+|EE4J PMC               |         |     |          |
+|Participant Members    |         |     |          |
+|Committer Members      |         |     |          |
```
Comment (kwsutter) [inline expression-language/5.0/_index.md:75]: Same comment as above about the missing reference to the mailing list ballot thread.

File: expression-language/5.0/_index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+* [Jakarta Expression Language 5.0 Release Record](https://projects.eclipse.org/projects/ee4j.el/releases/5.0.0)
+    * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
+
+# Plan
+
+The plan for the release is to tackle the open issues for the API component including:
+* Potential extensions to the API to support Jakarta Faces
+* Support for default methods
+* Potentially removing the dependency on the java.desktop JPMS module
+* Remove deprecated methods from API
```
Comment (kwsutter) [inline expression-language/5.0/_index.md:17]: Nice.  Good reason for updating the major version.  Thanks!

File: expression-language/5.0/_index.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+# Backwards compatibility
+Backward compatibility issue include:
+* Removal of deprecated methods
+* Removing the java.desktop dependency may create compatibility issues. The extent of those issues will be a factor in deciding whether to proceed with this or not.
```
Comment (scottmarlow) [inline expression-language/5.0/_index.md:22]: Is there an action expected to remove the `java.desktop` dependency?  I couldn't see any changes on the master branch related to "java.desktop".

For reference from the https://docs.oracle.com/javase/9/docs/api/java.desktop-summary.html description:
```
Defines the AWT and Swing user interface toolkits, plus APIs for 
accessibility, audio, imaging, printing, and JavaBeans.
```

Comment (markt-asf) [reply expression-language/5.0/_index.md:22]: https://github.com/eclipse-ee4j/el-ri/issues/47

I've only taken a quick look at it so far. There isn't a simple solution. It needs a wider a discussion in the community to figure out how we want to handle this.

**Pull request merged**